.github/workflows/build-pipeline.yml

```diff
@@ -347,41 +449,176 @@ jobs:
+        uses: actions/checkout@v3
+      - name: Cache build output
+        uses: actions/cache@v2
+        id: restore-build
+        with:
+          path: |
+            apps/**/build*
+            packages/**/build*
+          key: build-frontend-output-${{ github.sha }}
+      - name: Cache rebuild output
+        uses: actions/cache@v2
+        with:
+          path: |
+            .yarn/unplugged/
+          key: rebuild-output-${{ github.sha }}
+      - uses: actions/download-artifact@v2
+        with:
+          name: sls-package
+          path: 'sls-package'
+      - name: Deploy
+        run: |
+          yarn sls deploy --package sls-package --verbose
+
+  verify-sls:
+    needs: [setup, deploy-sls]
+    permissions:
+      packages: read
+    runs-on: ubuntu-latest
+    container:
+      image: ${{ needs.setup.outputs.integration-image-name }}
+      credentials:
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    env:
+      ASAP_API_URL: 'https://api-${{ needs.setup.outputs.pr-number }}-github.hub.asap.science'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Test
+        run: |
+          yarn test:e2e
+  e2e-test:
+    needs: [setup, deploy-sls]
+    runs-on: ubuntu-latest
+    container:
+      image: mcr.microsoft.com/playwright:focal
+    env:
+      ASAP_APP_URL: 'https://${{ needs.setup.outputs.pr-number }}-github.hub.asap.science'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Cache build output
         uses: actions/cache@v2
         id: restore-rebuild
         with:
           path: |
             .yarn/unplugged/
           key: rebuild-output-${{ github.sha }}
-      - name: Verify
+      - name: Install Playwright
         run: |
-          yarn schema:update
-          git diff --exit-code ./apps/asap-server/src/autogenerated-gql/
+          yarn workspace @asap-hub/e2e-tests run playwright install
+      - name: Test
+        run: |
+          yarn workspace @asap-hub/e2e-tests run playwright test
   notify_failure:
     runs-on: ubuntu-latest
     needs:
       [
         setup,
         test-unit,
         test-integration,
-        build,
-        rebuild,
+        build-frontend,
+        build-typecheck,
+        transpile,
+        typecheck,
         check,
         build-sls-package,
         bundlewatch,
         test-build-output,
         test-browser,
         prepare-squidex,
         verify-schema,
+        deploy-sls,
+        verify-sls,
+        e2e-test,
       ]
-    if: failure()
+    if: ${{ failure() }}
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
-      - uses: ./.github/actions/slack/
-        with:
-          message: 'Build Failed on GitHub: ${{ needs.setup.outputs.pr-number || github.head_ref }}'
-          webhook: ${{ secrets.SLACK_WEBHOOK }}
-          status: failure
+        uses: actions/checkout@v3
+      # - uses: ./.github/actions/slack/
```

Comment:
pretty sure we want this to happen no?

Reply:
I got asked to comment it out while we are developing, there was too much noise in the channel.

Reply:
I'll add it back, but we may need to re-think how and when we notify Slack.

Reply:
ahh i see, why is there so much noise though, i wonder? we do not notify in slack when the PR pipeline fails in gitlab, do we? in which case you probably want to remove it entirely, no?

Reply:
Removed!